src/parameterizations/vertical/MOM_set_diffusivity.F90

```diff
@@ -2053,7 +2201,7 @@ subroutine set_diffusivity_init(Time, G, GV, US, param_file, diag, CS, int_tide_
   type(diag_ctrl), target,  intent(inout) :: diag !< A structure used to regulate diagnostic output.
   type(set_diffusivity_CS), pointer       :: CS   !< pointer set to point to the module control
                                                   !! structure.
-  type(int_tide_CS),        pointer       :: int_tide_CSp !< Internal tide control structure
+  type(int_tide_CS),        intent(in), target :: int_tide_CSp !< Internal tide control structure
```

Comment:
This change causes the following error in one of our intel debug tests:

```
forrtl: severe (408): fort: (7): Attempt to use pointer INT_TIDE_CSP when it is not associated with a target
          
 mom_diabatic_driv        3523  MOM_diabatic_driver.F90
```

Reply:
I got confirmation from @raphaeldussin that we can revert this back to `pointer`.